src/node.cc

```diff
@@ -4556,7 +4557,9 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   {
     SealHandleScope seal(isolate);
     bool more;
+    PERFORMANCE_MARK(&env, LOOP_START);
     do {
+      PERFORMANCE_MARK(&env, LOOP_FRAME);
```

Comment:
Ah ok. I'm just looking through this PR now but does it introduce any work on idles? I played around with using an idle handle in the platform implementation and found that it slowed things down too much when I started benchmarking with acmeair. It could have just been the nature of the work I was doing but it might be good to get some initial idea of how these changes will impact performance.